app/helpers/save_this_page_helper.rb

```diff
@@ -2,8 +2,8 @@ module SaveThisPageHelper
   attr_reader :page_is_saved, :signed_in, :page_path
 
   PAGE_IS_SAVED_LINK_HREF = {
-    true => "/account/saved-pages/remove?page_path=",
-    false => "/account/saved-pages/add?page_path=",
+    true => "#{Plek.find('frontend')}/account/saved-pages/remove?page_path=",
+    false => "#{Plek.find('frontend')}/account/saved-pages/add?page_path=",
```

Comment:
This won't work in production because `Plek.find('frontend')` will give you `frontend.blue.whatever` whereas we'll need it to resolve to `www.whatever`